### DISCUSSION TITLE
Ironing top support interface

### DISCUSSION THREAD

**Discussion by TurboNimitz** [OrcaSlicer/OrcaSlicer#5750] [Ideas]
When printing with brake away supports like PETG on PLA, HiPS on Nylon.
Top Z distance = 0
Top Interface Spacing = 0
Adding ironing on top of this top interface (yellow) could generate a smoother surface for the planar overhang printing on top of this.
![image_2024-06-18_150247996](https://github.com/SoftFever/OrcaSlicer/assets/48719518/89b71ebb-e5e6-491e-b498-93f2580458a6)

**Comment by jpabuyer** [comment]
+1

CrealityPrint already does this, and it works wonderful. Supports are very easyly removable and leaves less marks

**Comment by atamgp** [comment]
this is already added in 2.3.1?
https://github.com/OrcaSlicer/OrcaSlicer/pull/9548

**Comment by atamgp** [comment]
there seems to be a bug though
https://github.com/OrcaSlicer/OrcaSlicer/issues/10840
